### PR TITLE
daemon: use an array for CRUSH_LOCATION

### PR DESCRIPTION
**Pull request by leseb**
Signed-off-by: Sébastien Han <seb@redhat.com>

### CODE REVIEW

File: ceph-releases/kraken/ubuntu/16.04/daemon/osd_scenarios/osd_common.sh [hunk 1]
```diff
@@ -14,7 +14,7 @@ start_osd() {
   else
     OSD_WEIGHT=$(df -P -k "$OSD_PATH" | tail -1 | awk '{ d= $2/1073741824 ; r = sprintf("%.2f", d); print r }')
   fi
-  ceph "${CLI_OPTS[@]}" --name=osd."${OSD_ID}" --keyring="$OSD_KEYRING" osd crush create-or-move -- "${OSD_ID}" "${OSD_WEIGHT}" "${CRUSH_LOCATION}"
+  ceph "${CLI_OPTS[@]}" --name=osd."${OSD_ID}" --keyring="$OSD_KEYRING" osd crush create-or-move -- "${OSD_ID}" "${OSD_WEIGHT}" "${CRUSH_LOCATION_DEFAULT[@]}"
```
Comment (mikegrass) [inline ceph-releases/kraken/ubuntu/16.04/daemon/osd_scenarios/osd_common.sh]: Should still be CRUSH_LOCATION, not CRUSH_LOCATION_DEFAULT, right?

Same for other files.

Comment (leseb) [inline ceph-releases/kraken/ubuntu/16.04/daemon/osd_scenarios/osd_common.sh]: Thanks!

**Comment by leseb** [comment]
jenkins test ceph_ansible-luminous-xenial-dedicated_journal

**Comment by leseb** [comment]
jenkins test ceph_ansible-jewel-xenial-cluster

**Pull request merged**